internal/store/alerts.go

```diff
@@ -30,9 +30,14 @@ type AlertLister interface {
 	Alerts(string, *atlas.ListOptions) ([]atlas.Alert, error)
 }
 
+type AlertAcknowledger interface {
+	Acknowledge(string, string, *atlas.AcknowledgeRequest) (*atlas.Alert, error)
```

Comment:
[aside] if we eventually implement the same for global alerts we may need to change this to `AcknowledgeAlert(...)` but for now I'm ok with this